src/app/public/modules/action-button/action-button-adapter-service.spec.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  Component,
+  ElementRef,
+  ViewChild
+} from '@angular/core';
+
+import {
+  ComponentFixture,
+  inject,
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  SkyActionButtonAdapterService
+} from './action-button-adapter-service';
+
+@Component({
+  selector: 'sky-test-cmp',
+  template: `
+    <div
+      [style.width]="parentWidth"
+    >
+      <div #el>
+        Hello world
+      </div>
+    </div>
+  `
+})
+class SkyActionButtonAdapterTestComponent {
+
+  public parentWidth: string;
+
+  @ViewChild('el', {
+    read: ElementRef,
+    static: true
+  })
+  public input: ElementRef;
+
+}
+
+describe('Description action button adapter service', () => {
+  let fixture: ComponentFixture<SkyActionButtonAdapterTestComponent>;
+  let inputRef: ElementRef;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        SkyActionButtonAdapterTestComponent
+      ],
+      providers: [
+        SkyActionButtonAdapterService
+      ]
+    });
+
+    fixture = TestBed.createComponent(SkyActionButtonAdapterTestComponent);
+    fixture.detectChanges();
+    inputRef = fixture.componentInstance.input;
+  });
+
+  it('should return width for parent', inject(
+    [SkyActionButtonAdapterService],
+    (adapter: SkyActionButtonAdapterService) => {
```

Comment:
You don't have to worry about this, just sharing. I think you might be able to use `TestBed.inject` to pull out dependencies in the future. I prefer it since it has less boilerplate to setup. 👍 
```
const adapter = TestBed.inject(SkyActionButtonAdapterService);
```

Reply:
Done.